emsdk.py

```diff
@@ -2584,10 +2591,9 @@ def construct_env(tools_to_activate):
         else:
           assert False
 
-      errlog(key + ' = ' + value)
+    errlog('')
```

Comment:
Is this added an extra empty line to the output?

Reply:
yeah, just to make formatting a bit nicer

Reply:
But why do you say this is nicer?   Right no I see the following output on my machine:

```
sbc.sbc-glaptop (~/dev/wasm/emsdk) ((53c184e...)) $ . ./emsdk_env.sh 
Adding directories to PATH:
PATH += /home/sbc/dev/wasm/emsdk
PATH += /home/sbc/dev/wasm/emsdk/upstream/emscripten
PATH += /home/sbc/dev/wasm/emsdk/node/12.18.1_64bit/bin

Setting environment variables:
EMSDK = /home/sbc/dev/wasm/emsdk
EM_CONFIG = /home/sbc/dev/wasm/emsdk/.emscripten
EM_CACHE = /home/sbc/dev/wasm/emsdk/upstream/emscripten/cache
EMSDK_NODE = /home/sbc/dev/wasm/emsdk/node/12.18.1_64bit/bin/node
sbc.sbc-glaptop (~/dev/wasm/emsdk) ((53c184e...)) $
```

To me an extra empty line at the end would look pointless/messy.. but maybe that is just me?